container/database.go

```
@@ -1,18 +1,17 @@
-package db
+package container
 
 import (
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
 	"log"
 )
 
-var Instance *gorm.DB
-var dbError error
-
-func Connect(connectioString string) {
-	Instance, dbError = gorm.Open(postgres.Open(connectioString), &gorm.Config{})
+func GetDbConnection() *gorm.DB {
+	Instance, dbError := gorm.Open(postgres.Open("postgres://postgres:admin@localhost:5432/anywrstudentproject"), &gorm.Config{})
 	if dbError != nil {
 		log.Fatal("error configuring the database: ", dbError)
```

when we have an error on the DB connection we should add a panic 

we should raise the error so that the project cannot run without having a correct connection to db

---

if any problem with the db connection -> all apps will not run, what i should do other handling error?